contracts/contracts/RequestManager.sol

```diff
@@ -119,17 +123,23 @@ contract RequestManager is Ownable, LpWhitelist, RestrictedCalls, Pausable {
         address stakeRecipient
     );
 
-    event FinalityPeriodUpdated(uint256 targetChainId, uint256 finalityPeriod);
-
     /// Emitted when token object of a token address is updated.
     ///
     /// .. seealso:: :sol:func:`updateToken`
     event TokenUpdated(
         address tokenAddress,
         uint256 transferLimit,
-        uint96 minLpFee,
-        uint32 lpFeePPM,
-        uint32 protocolFeePPM
+        uint256 ethInToken
+    );
```

Comment:
While we are here, I think it would be nice to make `address tokenAddress` `indexed`.